### PR TITLE
[LRA-19] Login page template

### PR DESCRIPTION
**Pull request by Shalin21**
feat(Login page template): Create login template
feat(Formik): Create new types of field -
password and checkbox

Image:
Screenshot 2020-01-09 at 22 05 10 (2)

### CODE REVIEW

File: src/Pages/Login/loginPage.tsx [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import React from 'react';
+import { LoginPageLayout } from '../../layouts/LoginPageLayout/LoginPageLayout'
 
-export const loginPage = function() {
-    return <h1>Login</h1>  ;
+export const loginPage = () => {
```
Comment (mkozachok) [inline src/Pages/Login/loginPage.tsx]: `export const loginPage = () =><LoginPageLayout />`

File: src/components/forms/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+} from "./styled";
+import {
+  AntInput,
+  AntPassword,
+  AntCheckbox
+} from "../../../components/formControls";
+import { SpinBlock } from "../UpdateProductForm/styled";
+
+export const LoginForm = () => {
+  const [isLoading, setIsLoading] = useState(false);
```
Comment (mkozachok) [inline src/components/forms/LoginForm/LoginForm.tsx]: `setIsLoading` is not using

File: src/components/forms/LoginForm/LoginForm.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    rememberMe: false
+  };
+
+  return !isLoading ? (
+    <div
+      style={{
+        position: "relative"
+      }}
+    >
+      <FormBlock></FormBlock>
```
Comment (mkozachok) [inline src/components/forms/LoginForm/LoginForm.tsx:32]: Why we need this `<FormBlock></FormBlock>`

File: src/components/forms/LoginForm/LoginForm.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            </RowDiv>
+            <StyledButton
+              htmlType="submit"
+              type="primary"
+              loading={isSubmitting}
+              disabled={!isValid || isSubmitting}
+            >
+              Log in
+            </StyledButton>
+            <RowDiv>
```
Comment (mkozachok) [inline src/components/forms/LoginForm/LoginForm.tsx:74]: Move to separate component

**Review by mkozachok — CHANGES_REQUESTED** [review]

File: src/components/forms/LoginForm/LoginForm.tsx (hunk 3 above)
Comment (Shalin21) [reply src/components/forms/LoginForm/LoginForm.tsx:32]: It's used for background

File: src/Pages/Login/loginPage.tsx (hunk 1 above)
Comment (Shalin21) [reply src/Pages/Login/loginPage.tsx]: Fixed

File: src/components/forms/LoginForm/LoginForm.tsx (hunk 2 above)
Comment (Shalin21) [reply src/components/forms/LoginForm/LoginForm.tsx]: Fixed

File: src/components/forms/LoginForm/LoginForm.tsx (hunk 4 above)
Comment (Shalin21) [reply src/components/forms/LoginForm/LoginForm.tsx:74]: Done

**Pull request merged**